hw/ip/edn/data/edn.hjson

```diff
@@ -259,6 +259,59 @@
                 '''
         }
       ]
-    }
+    },
+    {
+      name: "ERR_CODE",
+      desc: "Hardware detection of error conditions status register",
+      swaccess: "rw",
+      hwaccess: "hwo",
+      fields: [
+        { bits: "0",
+          name: "SFIFO_RESCMD_ERR",
+          desc: '''
+                This bit will be set to one when an error has been detected for the
+                reseed command FIFO. The type of error is reflected in the type status
+                bits (bits 28 through 30 of this register).
+                This bit will stay set until firmware clears it.
+                '''
+        }
+        { bits: "1",
+          name: "SFIFO_GENCMD_ERR",
+          desc: '''
+                This bit will be set to one when an error has been detected for the
+                generate command FIFO. The type of error is reflected in the type status
+                bits (bits 28 through 30 of this register).
+                This bit will stay set until firmware clears it.
+                '''
+        }
+        { bits: "28",
+          name: "FIFO_WRITE_ERR",
+          desc: '''
+                This bit will be set to one when any of the source bits (bits 0 through 1 of ths
+                this register) are asserted as a result of an error pulse generated from
```

Comment:
Nit: Spare "ths"

Reply:
@rswarbrick thanks for the re-review! Spelling errors fixed. Current RTL is auto-gen'ed, so not sure this is ever really reviewed.